legislativas/20220130_legislativas/BE_31122021.md

```diff
@@ -0,0 +1,2959 @@
+# Razões fortes, compromissos claros
+
+Saúde | Salário | Clima
+Programa eleitoral 2022-2026
+
+## Introdução / A esquerda tem programa
```

Comment:
Troca este e os restantes `##` por `#`. Para aparecer como secções no site: https://www.politicaparatodos.pt/party/BE/manifesto.

Não te esqueças de baixar um nível aos headers intermédios.

Reply:
Mantenho o primeiro título como `h1`?

Quais os níveis relevantes para aparecer na navegação: 1 e 2 apenas?

Reply:
Acabei por remover para todos um nível de heading.

Reply:
É exatamente o que fizeste. Thanks